src/index.ts

```diff
@@ -3,7 +3,7 @@
 import type { IAutoPollOptions, IConfigCatLogger, ILazyLoadingOptions, IManualPollOptions } from "configcat-common";
 import type { GetValueType, WithConfigCatClientProps } from "./ConfigCatHOC";
 import withConfigCatClient from "./ConfigCatHOC";
-import { useConfigCatClient, useFeatureFlag } from "./ConfigCatHooks";
+import { useConfigCatClient, useConfigCatClientByConfigId, useFeatureFlag, useFeatureFlagByConfigId } from "./ConfigCatHooks";
```

Comment:
As I understand, these functions have been removed.

```suggestion
import { useConfigCatClient, useFeatureFlag } from "./ConfigCatHooks";
```